ImageItem/ImageItem.module.less

```diff
@@ -11,7 +11,14 @@
 		}
   	}
 
-  	.applySkins({
+	&.captionOverImage {
+		.caption {
+			overflow: hidden;
```

Comment:
What is our UX/GUI for too long texts? Clipping it, adding `...`, or something else?

Reply:
For now we don't have UX/GUI design when text is long. I went with this approach, but it can be changed if required.

Reply:
I got it. Let's support it separately if needed later.